debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1881,9 +1884,11 @@ public void stopInTheMiddleOfTxAndRestart() throws Exception {
     }
 
     @Test
-    @SkipWhenDecoderPluginNameIsNot(value = SkipWhenDecoderPluginNameIsNot.DecoderPluginName.PGOUTPUT, reason = "Tests specifically that pgoutput gracefully skips these messages")
-    public void shouldGracefullySkipTruncateMessages() throws Exception {
-        startConnector();
+    @SkipWhenDatabaseVersion(check = EqualityCheck.LESS_THAN, major = 11, reason = "TRUNCATE events only supported in PG11+ PGOUTPUT Plugin")
```

Comment:
So wal2json does not support truncate events?

Reply:
Yes. Only `PG11+ PGOUTPUT` plugin supports Truncate events.

Reply:
Ok. Can you log a follow-up issue for us to add TRUNCATE support for decoderbufs, too?